### PR TITLE
Adding mpcmd

### PR DESCRIPTION
**Pull request by dotmet**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/mpcmd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    - tests
+  commands:
+    - python tests/test_import.py
+    
+about:
+  home: https://github.com/dotmet/mpcmd
+  summary: 'Extensible mpcd fluid simulation package'
+  description: |
+    An extensible fluid simulation package which combines multiple MPCD and md methods.
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
```
Comment (synapticarbors) [inline recipes/mpcmd/meta.yaml:42]: Please remove comments that are not specific to this package

File: recipes/mpcmd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+package:
+  name: mpcmd
+  version: 0.0.5
+
+source:
+  url: https://files.pythonhosted.org/packages/bf/8c/07e22c42458f6dac37409979358b366cdf0c242bf648183cbdab6fba8eb5/mpcmd-0.0.5.tar.gz
+  sha256: b95b11b6465f73fcf4cbd095b873cac0140471b3ee1160ae980f4699bd08c302
+
+build:
```
Comment (synapticarbors) [inline recipes/mpcmd/meta.yaml:9]: It looks like this package should be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

File: recipes/mpcmd/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+package:
+  name: mpcmd
+  version: 0.0.5
+
+source:
+  url: https://files.pythonhosted.org/packages/bf/8c/07e22c42458f6dac37409979358b366cdf0c242bf648183cbdab6fba8eb5/mpcmd-0.0.5.tar.gz
```
Comment (synapticarbors) [inline recipes/mpcmd/meta.yaml:6]: Please use the canonical url shown in the example `meta.yaml`:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L16

File: recipes/mpcmd/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    - python
+    - numpy
+    - matplotlib-base
+    - scipy
+    - numba
+    - gsd
+    - memory_profiler
+    - seaborn
+
+test:
```
Comment (synapticarbors) [inline recipes/mpcmd/meta.yaml:29]: We encourage adding a `pip check` when possible:

https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L61-L64

**Review by synapticarbors — CHANGES_REQUESTED** [review]

**Comment by dotmet** [comment]
@synapticarbors Sorry, i am not familiar with the procedure, there are some incorrect operations. I will rewrite some parts of this package using cpp, so I decide to use conda-forge channel. I will check the reviews and update the yaml file tomorrow. Thanks :)

**Comment by dotmet** [comment]
@conda-forge/help-python ready for review

**Pull request closed without merging**